packages/demos/src/todomvc/webpack-config.js

```diff
@@ -1,46 +1,57 @@
-import CopyPlugin from 'copy-webpack-plugin';
-import {join} from 'path';
+// @ts-check
+const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
 // @ts-ignore
-import postcssPresetEnv from 'postcss-preset-env';
-import {Configuration} from 'webpack';
-import merge from 'webpack-merge';
-import {webpackBaseConfig} from '../webpack-base-config';
+const postcssPresetEnv = require('postcss-preset-env');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const webpackBaseConfig = require('../webpack-base-config');
 
-const configs: Configuration[] = [
+const websiteBuildDir = path.resolve(
```

Comment:
rename to `websiteDirname` or `websiteBuildDirname`

Reply:
done